src/Helpers/Excerpt.php

```diff
@@ -73,8 +74,14 @@ public function getExcerpt($length = 200, $includeTitle = false, $focus = null)
 
             $excerpt = '';
             array_walk($this->body, function ($value, $key) use (&$excerpt, $stripKeys) {
-                if (is_string($value) && !in_array($key, $stripKeys)) {
-                    $excerpt .= $value . ' ';
+                if (in_array($key, $stripKeys)) {
+                    return;
+                }
+                // We need non-empty strings that don't look like serialized JSON.
+                // Otherwise, Twig Markup is also OK.
+                if ((is_string($value) && !empty($value) && ($value[0]!='{')) ||
```

Comment:
Perhaps we should also check for `($value[0] !== '[')` (some extensions store json arrays IIRC), and strict comparison would be nice even though it probably won't make a difference.

Reply:
Good points both. Will do.